src/lib/userStorage/UserProfileStorage.js

```diff
@@ -424,13 +448,23 @@ export class UserProfileStorage implements ProfileStorage {
     return getUserModel(displayProfile)
   }
 
-  getFieldPrivacy(field: string): string {
+  /**
+   * Returns field privacy
+   * @param field
+   * @returns {FieldPrivacy}
+   */
+  getFieldPrivacy(field: string): FieldPrivacy {
     const currentPrivacy = this.profile[field]?.privacy
 
     return currentPrivacy || this.profileSettings[field].defaultPrivacy || 'public'
   }
 
-  async validateProfile(profile: any): Promise<{ isValid: boolean, errors: {} }> {
+  /**
+   * Checks if profile is valid profile
+   * @param profile
+   * @returns {Promise<{isValid: boolean, errors: {}}|{isValid, errors}>}
+   */
+  async validateProfile(profile: Profile): Promise<{ isValid: boolean, errors: {} }> {
```

Comment:
please ask @sirpy about this task: https://app.zenhub.com/workspaces/gooddollar-5d50833888a83c6880d3e345/issues/gooddollar/gooddapp/3385
maybe we could do it in the scope of this refactoring (just data-validation part via `yup`, without refactoring UI forms via `formik`)

Reply:
As we decided on tech sync. We will leave this for now